### ISSUE TITLE
Bug: Can't download an specific version

### ISSUE THREAD

**Issue by tdreid** [facundovictor/jsfiddle-downloader#4]
It seems that if I run...

```
	jsfiddle-downloader -l https://jsfiddle.net/[auser]/[acode]/10/ -o dlFiddle.html
```

...jsfiddle-downloader will download whatever version is set to base .  It'd be neat to download the version to which the input url points.

**Comment by facundovictor** [comment]
Thanks for reporting it @tdreid!

**Comment by facundovictor** [comment]
Resolved in **v0.1.3**
